utils/validate-quickstart-data-source.ts

```diff
@@ -0,0 +1,68 @@
+import {
```

Comment:
I could see some unit tests being helpful for this file. Especially if it's going to pull from nerdgraph (see other comment!)